mpc/README.md

```diff
@@ -81,9 +84,10 @@ Digest written to: response.bin.digest
 ...
 ```
 
-For phase2:
+For phase2 (assuming `mpc-client-phase2` is in the `PATH`):
```

Comment:
I'm guessing that you hit an issue when the `mpc-coord-phase2` tool was not available?

Rather than users adding this to their path, it might be better for UX to update the code that looks for the tool in the default location (e.g. check for either `mpc-coord-phase2` or `mpc-client-phase2`, or use a flag to set client/server). This will avoid the user having to update his local environment, in most cases.